habitat/plan.sh

```diff
@@ -33,17 +34,9 @@ pkg_deps=(
   core/libarchive
 )
 
-pkg_svc_user=root
-
-do_before() {
-  do_default_before
-  update_pkg_version
-}
```

Comment:
This whole `do_before` wasn't doing anything, I feel it was a copy/paste from other plans, maybe.

Reply:
We should be using the `do_before` to load the package version rather than `cat`-ing it in the `pkg_version` declaration.